src/main/java/com/exasol/adapter/dynamodb/dynamodbmetadata/AbstractDynamodbIndex.java

```diff
@@ -4,20 +4,20 @@
 import java.util.Optional;
 
 /**
- * This class represents a DynamoDB index. This can either be the default index of a table or a secondary index.
+ * This class represents a DynamoDB index. This can either be the primary index of a table or a secondary index.
```

Comment:
I think the description should be a little bit different for an abstract class. Something like "This class is an abstract base for..."